packages/node_modules/@ciscospark/internal-plugin-reaction/README.md

```diff
@@ -0,0 +1,108 @@
+# @ciscospark/internal-plugin-reaction
+
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
+
+> Plugin for the Reaction service
+
+This is an internal Cisco Spark plugin. As such, it does not strictly adhere to semantic versioning. Use at your own risk. If you're not working on one of our first party clients, please look at our [developer api](https://developer.ciscospark.com/getting-started.html) and stick to our public plugins.
+
+- [Install](#install)
+- [Usage](#usage)
+- [Contribute](#contribute)
+- [Maintainers](#maintainers)
+- [License](#license)
+
+## Install
+
+```bash
+npm install --save @ciscospark/internal-plugin-reaction
+```
+
+## Usage
+
+
+Wiki design: https://sqbu-github.cisco.com/WebExSquared/cloud-apps/wiki/Reactions(Design-&-Api-Spec)
+
+```js
+
+import '@ciscospark/internal-plugin-reaction';
+
+import SparkCore from '@ciscospark/spark-core';
+
+const spark = new SparkCore();
+spark.internal.reaction.WHATEVER
+
+```
+
+
+1. Add Reaction:
+
+
+  The only input will be an options variable. The required values to be passed in this object are
+  ```
+  {
+    participant,
+    reactionList,
+    actID,
+    convoID,
+    targetURL
+  }
+  ```
+
+  the function will then call spark.request with
+  ```
+    {
+      method: 'POST',
+      service: 'conversation',
+      resource: '/activities',
+      options,
+      body:
+      {
+        clientTempId: uuid.v4(),
+        objectType: 'activity',
+        verb: 'add',
+        actor: {
+          objectType: 'person',
+          id: options.participant.id,
+          entryUUID: options.participant.entryUUID
+        },
+        object: {
+          objectType: 'messageProperty',
+          tags: options.reactionList,
+          id: options.actID,
+          reactionType: ''
+        },
+        target: {
+          id: options.convoID,
+          url: options.targetURL,
+          objectType: 'conversation'
+        }
+      }
+    }
+  ```
+
+
+
+2. Delete reaction
```

Comment:
Before going too far into it, might be helpful to do some basic docs and showing mock usage of each of these.

Reply:
I flushed out the mock usage for add and delete